packages/manager/apps/web/client/app/private-database/cpu-throttle/private-database-cpu-throttle.controller.js

```diff
@@ -0,0 +1,57 @@
+import clone from 'lodash/clone';
+
+angular.module('App').controller(
+  'PrivateDatabaseCpuThrottleCtrl',
+  class PrivateDatabaseCpuThrottleCtrl {
```

Comment:
Not possible to use `export` syntax here ?

Reply:
This code is actually made to be iso with the code that is managing the OOM. I would prefer to keep the codebase coherent.